cellar-tags/tagging.md

```diff
@@ -167,9 +167,9 @@ i.e. [@!RFC5646] two-letter region subtags, without the UK exception.
 
 ## Target Types
 
-The `TargetTypeValue` element allows tagging of different parts that are inside or outside a
+The `TargetTypeValue` element allows tagging of different parts that are within or outside a
 given file. For example, in an audio file with one song you could have information about
-the album it comes from the CD set even if it's not found in the file.
+the album it comes from the CD set even if the whole CD set is not found in the file.
```

Comment:
```suggestion
the album or CD set it comes from, even if the whole album or CD set is not found in the file.
```
I'm guessing at my suggestion here, but the current sentence isn't super readable IMO.